src/Shared/Traits.cs

```diff
@@ -50,6 +50,11 @@ public Traits()
     internal class EscapeHatches
     {
         /// <summary>
+        /// Force whether Project based evaluations should evaluate elements with false conditions.
+        /// </summary>
+        public readonly bool? EvaluateElementsWithFalseConditionInProjectEvaluation = ParseNullableBoolFromEnvironmentVariable(nameof(EvaluateElementsWithFalseConditionInProjectEvaluation));
```

Comment:
Should this start with MSBUILD to match the other flags?

Reply:
I guess it would be nicer to respect convention. I'll change it.
My intent was to reduce the amount of information, by keeping the member name the same as the environment variable.